core/yarn.lock

```diff
@@ -318,10 +318,10 @@
   resolved "https://registry.yarnpkg.com/@bisonai/orakl-contracts/-/orakl-contracts-1.3.1.tgz#4d309d6d94327f42168574cbffa033fdcfc12a1d"
   integrity sha512-HCIwvIkjlGA+izRtmVPT0ZlP7WFpkAwbvxeMBqq3tAtxSG/+EcsM7K4T1e/8QpmHzUmLDgvGDrqE8CTtp5bJRA==
 
-"@bisonai/orakl-vrf@^0.1.1":
-  version "0.1.1"
-  resolved "https://registry.yarnpkg.com/@bisonai/orakl-vrf/-/orakl-vrf-0.1.1.tgz#61e90a44520a7d9abf1a20c2c863201742c0b2e2"
-  integrity sha512-mz1DhGQEgKXwk78ZAe9EG1C9tat/fqAch13K7muK2r1xCSAVMoFJU0II39HbH99cC2mz6/l6I3B0szeyrMw+Zw==
+"@bisonai/orakl-vrf@^0.2.0":
```

Comment:
@KelvinThai Was there something missing in the older version?

Reply:
the 0.2.0 is the newer and includes the L1 and L1Endpoint. However we're still changing it so it does not make a big difference, just try to use the newest version here.

Reply:
@KelvinThai I think I was confused before. `@bisonai/orakl-vrf` does not include any on-chain implementation, it is only for computation of VRF off-chain. Let's not change it in this PR.

https://github.com/Bisonai/orakl/tree/master/vrf